dashboard/public/openapi.yaml

```diff
@@ -23,10 +20,1621 @@ info:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/Release'
+        '401':
+          description: Unauthorized
+    delete:
+      tags:
+        - kubeops
+      summary: 'DeleteRelease'
+      description: ''
+      operationId: 'DeleteRelease'
+      parameters:
+        - name: cluster
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Cluster name
+        - name: namespace
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Namespace name
+        - name: releaseName
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Release name
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: string
+        '401':
+          description: Unauthorized
+    put:
+      tags:
+        - kubeops
+      summary: 'OperateRelease'
+      description: ''
+      operationId: 'OperateRelease'
+      parameters:
+        - name: cluster
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Cluster name
+        - name: namespace
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Namespace name
+        - name: releaseName
+          in: path
+          required: true
+          schema:
+            type: string
+            default: default
+          description: Release name
+        - name: action
+          in: query
+          schema:
+            type: string
+            enum: ["upgrade","rollback"]
+            default: upgrade
+          description: Action to perform
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/Release'
+        '401':
+          description: Unauthorized
+  
 components:
+  schemas:
+    # pkg/proxy/proxy.go
+    AppOverview:
+      type: object
+      properties:
+        nareleaseNameme:
+          type: string
+        version:
+          type: string
+        namespace:
+          type: string
+        icon:
+          type: string
+        status:
+          type: string
+        chart:
+          type: string
+        chartMetadata:
+          $ref: '#/components/schemas/ChartMetadata'
+    # /pkg/mod/k8s.io/helm@v2.16.0+incompatible/pkg/proto/hapi/chart/metadata.pb.go
```

Comment:
Why is this based on Helm 2.6 instead of Helm 3 (as in https://github.com/kubeapps/kubeapps/pull/2291/files#diff-da58234bb539b4febe5d392f206d2fd4025fb7c76f89c71653c4d590bcd72f2bR1474)?

Reply:
To elaborate on this, the generator broke on this definition.
I noticed that the actual payload is a little bit different, here is the diff I applied locally:
```
diff --git a/kubeapps-openapi.yaml b/kubeapps-openapi.yaml
index 5ce7b11..76bde29 100644
--- a/kubeapps-openapi.yaml
+++ b/kubeapps-openapi.yaml
@@ -1410,7 +1410,7 @@ components:
     AppOverview:
       type: object
       properties:
-        nareleaseNameme:
+        releaseName:
           type: string
         version:
           type: string
@@ -1429,7 +1429,9 @@ components:
       type: object
       properties:
         annotations:
-          type: string
+          type: object
+          additionalProperties:
+            type: string
         apiVersion:
           type: string
         appVersion:
@@ -1445,9 +1447,7 @@ components:
         home:
           type: string
         icon:
-          type: array
-          items:
-            type: string
+          type: string
         kubeVersion:
           type: string
         maintainers:

```

I guess some definitions changed from Helm 2 to Helm 3... Hope this is correct and useful 😄